definitions/types/cidr-private-networks.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "title": "Private Network CIDR",
+    "type": "string",
+    "description": "Enter a CIDR IP address range to use for the size of your virtual network.",
+    "pattern": "^(?:10\\.(?:[0-9]|[0-9]{2}|1[0-9][0-9]|2[0-4][0-9]|25[0-5])|172\\.(?:1[6-9]|2[0-9]|3[0-1])|192\\.168)(?:\\.(?:[0-9]|[0-9]{2}|1[0-9][0-9]|2[0-4][0-9]|25[0-5])){2}(?:\/(?:[8-9]|1[0-9]|20))?$",
```

Comment:
This pattern specifically is only the private network space.
Additionally, it only allows masks from 8-20. Most networks allow out to 24, and most subnets allow out to 28. The idea is we consolidate around all our networks being no smaller than `/20`. There is no real point to having a `/24` network that is only ~250 addresses. Technically this actually breaks AWS. So either every network has its own pattern, or we use this across all our top-level networks, and can adjust down the road if we need more customization per cloud.